synctl

```diff
@@ -1597,6 +1618,29 @@ class PatchSyntheticTest(SyntheticTest):
             print(not_found_e)
             sys.exit(-1)
 
+    def patch_script_file(self, script_file, test_id):
+        """--entry-file update scriptFile """
+        test_result = self.retrieve_a_synthetic_test(test_id)
+        try:
+            test_bundle = test_result[0]["configuration"]["scripts"]["bundle"]
+            payload = {
+                "configuration":
+                    {
+                        "syntheticType": "HTTPScript",
+                        "scripts": {
+                            "bundle": test_bundle,  # required
+                            "scriptFile": ""  # required
+                        }
+                    }
+            }
+            if script_file is None:
+                print("script_file should not be None")
```

Comment:
I tried this command with options `--entry-file  "" `, it can run successfully, and `--entry-file` should not be empty string.
```
synctl patch test hLTHXvzHg8KDiO5hJV3y --entry-file  "" --use-env pink  
hLTHXvzHg8KDiO5hJV3y updated
```